nvdaHelper/local/UIAUtils.cpp

```diff
@@ -18,3 +18,9 @@ PROPERTYID registerUIAProperty(GUID* guid, LPCWSTR programmaticName, UIAutomatio
 	registrar->Release();
 	return propertyId;
 }
+
+int registerUIAAnnotationType(GUID* guid) {

```

Comment:
Shouldn't this go in NVDAHelperLocalWin10, as that's basically a WinRT interaction library?

Reply:
From a usage point of view, I think it makes sense that registerUIAAnnotationType is next to registerUIAProperty. True they don't necessarily use the same technology though. 
I think though nvdaHelperLocalWin10 currently is all C++/CX? where as winRT is actually all pure c++. So unless there really is a technical reason stopping this from being compiled, I think I would prefer to leave it in nvdaHelperLocal.

Reply:
I'm fine with that. Note that #11768 intends to switch everything to pure C++ WinRT